custom_model_runner/datarobot_drum/drum/gpu_predictors/nim_predictor.py

```diff
@@ -41,6 +41,17 @@ def __init__(self):
         self.max_model_len = self.get_optional_parameter("NIM_MAX_MODEL_LEN")
         self.log_level = self.get_optional_parameter("NIM_LOG_LEVEL")
 
+        # DR prefix to denote that this is not some standard NIM variable of any sort.
+        self.health_route = self.get_optional_parameter(
+            "DR_NIM_HEALTH_ROUTE", default_value=self.DEFAULT_HEALTH_ROUTE
+        )
+        self.openai_port = self.get_optional_parameter(
+            "DR_NIM_SERVER_PORT", default_value=self.openai_port
```

Comment:
There is an attempt.. to have a standard NIM_SERVER_PORT, which should work on the NIM side, but as always its inconsistent.
Anyway, these vars are for DRUM side, not NIM